diff --git a/node_modules/rn-emoji-keyboard/lib/module/components/ConditionalContainer.js b/node_modules/rn-emoji-keyboard/lib/module/components/ConditionalContainer.js
index b11e510..865551c 100644
--- a/node_modules/rn-emoji-keyboard/lib/module/components/ConditionalContainer.js
+++ b/node_modules/rn-emoji-keyboard/lib/module/components/ConditionalContainer.js
@@ -1,5 +1,6 @@
 import React from 'react';
-import { SafeAreaView, View } from 'react-native';
+import { View } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
 export const ConditionalContainer = _ref => {
   let {
     children,
diff --git a/node_modules/rn-emoji-keyboard/lib/module/components/EmojiStaticKeyboard.js b/node_modules/rn-emoji-keyboard/lib/module/components/EmojiStaticKeyboard.js
index 505c217..77f8934 100644
--- a/node_modules/rn-emoji-keyboard/lib/module/components/EmojiStaticKeyboard.js
+++ b/node_modules/rn-emoji-keyboard/lib/module/components/EmojiStaticKeyboard.js
@@ -1,6 +1,7 @@
 function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
 import * as React from 'react';
-import { StyleSheet, View, Animated, SafeAreaView, Platform } from 'react-native';
+import { StyleSheet, View, Animated, Platform } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
 import { EmojiCategory } from './EmojiCategory';
 import { KeyboardContext } from '../contexts/KeyboardContext';
 import { Categories, CATEGORY_ELEMENT_WIDTH } from './Categories';
